Client/Frontend/Home/ReaderPanel.swift

```diff
@@ -3,51 +3,52 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 import UIKit
-
 import Storage
 
-class ReaderPanel: UITableViewController {
-    var profile: Profile!
+class ReaderPanel: UITableViewController, HomePanel {
 
-    private var sites = [Site]()
-    /*
-        Site(title:"Royals Sweep Orioles to Advance to World Series", url:"http://www.nytimes.com/2014/10/16/sports/baseball/royals-keep-rolling-and-advance-to-the-world-series.html?hp&action=click&pgtype=Homepage&version=LargeMediaHeadlineSum&module=photo-spot-region&region=top-news&WT.nav=top-news&_r=0"),
-        Site(title:"How Not to Be Fooled by Odds", url:"http://www.nytimes.com/2014/10/16/upshot/how-not-to-be-fooled-by-odds.html?hp&action=click&pgtype=Homepage&version=HpSum&module=second-column-region&region=top-news&WT.nav=top-news"),
-        Site(title:"Against Rules, Amber Vinson, Dallas Worker With Ebola, Boarded Plane", url:"http://www.nytimes.com/2014/10/16/us/ebola-outbreak-texas.html?hp&action=click&pgtype=Homepage&version=LedeSum&module=first-column-region&region=top-news&WT.nav=top-news")
-    ]
-    */
+    var readingListItemsCursor: Cursor? = nil
```

Comment:
You could make this non-optional and instead just set it to an empty cursor when its not assigned. Those can even have some "state", so we could do things like "Loading".